src/Devolutions.AvaloniaTheme.MacOS/Controls/TextBox.axaml

```diff
@@ -106,7 +105,7 @@
       <Setter Property="CaretBrush" Value="{DynamicResource SystemAccentColor}" />
       <Setter Property="FontSize">
          <Setter.Value>
-            <MultiBinding Converter="{x:Static c:DevoMultiConverters.RevealPasswordToFontSizeConverter}">
+            <MultiBinding Converter="{x:Static DevoMultiConverters.RevealPasswordToFontSizeConverter}">
```

Comment:
Should be removed everywhere ...

Reply:
I thought I did a pass to remove them all (converters, markup extensions and controls). 
I'll go take another look to make sure I didn't miss any. It would still work perfectly fine, it's just for consistency / simplifying the codebase!